docs/install/any-kubernetes-cluster.md

```diff
@@ -17,36 +17,17 @@ aliases:
   - /docs/install/knative-with-minikube
   - /docs/install/knative-with-minishift
   - /docs/install/knative-with-pks
+showlandingtoc: "false"
 ---
 
-This guide walks you through the installation of the latest version of Knative.
-Note if you are upgrading an existing installation, follow the
-[instructions here](./upgrade-installation.md).
+You can install Knative by applying YAML files using the `kubectl` CLI.
 
-Knative has two components, which can be installed and used independently or
```

Comment:
Checking this is intentional: I think this drops the fact that Serving and Eventing can be installed independently, which means if you just wanted to install (e.g.) serving, the instructions are now effectively twice as long. I _think_ quite a few people only want one or the other. Do we intend to drop this fact here?

Reply:
If we're going to have it, we should move it up to `install/_index.md`. I think it's an important call-out, so I'll move it up there.

Reply:
Agreed @evankanderson - it's actually called out on the main landing page because I agree it's important (https://knative.dev/docs/) but I guess my thinking here was just trying to remove fluff where possible; we're planning to split this into two sections for installing serving and eventing separately soon anyway. I'll add it to the index though in another PR since this one already got merged :confused:

Reply:
It's in the section above now https://github.com/knative/docs/pull/3297/files/17839389f802f56108e13a19938374327b61891d#diff-8ff3dcba7bbec8163f1d3589ca5a14ed90107bc5b31715ff4f79649f478970e2R8

It's also called out on the main landing page